### DISCUSSION TITLE
See other Forems on the same theme

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#17434] [Icebox]
Why not link other Forem with a tag if possible? For example, the VSCode tag fits well in the theme of [the Forem VSCodeTips](https://community.vscodetips.com/).

The best thing would be to have the title of the Forem on the right side of the picture, but if there are a lot of Forems, either "See more Forems in the same category" or just the pictures. 

![See other Forems on the same theme](https://user-images.githubusercontent.com/14293805/165308862-4611688d-d83d-4e22-850b-8ab4987886f6.png)

**Comment by amywtlin** [comment]
@thomasbnt This looks like a great idea that helps users discover relevant Forems! We don't have anything planned yet on the horizon, but I am putting your request in `icebox` so we can revisit it at a later time.

**Comment by amywtlin** [comment]
Also, to double confirm that I understand your request correctly, you meant that there should be a way to configure relevant Forems on a tag page on DEV, so that people can join Forems of topics related to their interests. Is that correct?

**Reply by thomasbnt** [reply]
Exactly!
